### PR TITLE
chore: update contrib modules

### PR DESCRIPTION
**Pull request by dyladan**

### CODE REVIEW

File: detectors/node/opentelemetry-resource-detector-github/package.json [hunk 1]
```diff
@@ -60,8 +60,8 @@
     "typescript": "4.1.3"
   },
   "dependencies": {
-    "@opentelemetry/api": "^0.15.0",
-    "@opentelemetry/core": "^0.15.0",
-    "@opentelemetry/resources": "^0.15.0"
+    "@opentelemetry/api": "^0.16.0",
+    "@opentelemetry/core": "^0.16.0",
```
Comment (Flarna) [inline detectors/node/opentelemetry-resource-detector-github/package.json:64]: it looks strange to have `"@opentelemetry/core": "^0.16.0"` here and `"@opentelemetry/core": "0.16.0"` in dev dependencies.
is there a reason why we use fixed versions in dev dependencies but not in prod dependencies?

File: examples/dns/package.json [hunk 2]
```diff
@@ -26,12 +26,12 @@
...
   "dependencies": {
-    "@opentelemetry/api": "^0.15.0",
-    "@opentelemetry/exporter-jaeger": "^0.15.0",
-    "@opentelemetry/exporter-zipkin": "^0.15.0",
-    "@opentelemetry/node": "^0.15.0",
+    "@opentelemetry/api": "^0.16.0",
+    "@opentelemetry/exporter-jaeger": "^0.16.0",
+    "@opentelemetry/exporter-zipkin": "^0.16.0",
+    "@opentelemetry/node": "^0.16.0",
     "@opentelemetry/plugin-dns": "^0.13.0",
```
Comment (Flarna) [inline examples/dns/package.json:33]: Is the update to 0.14.0 a followup PR?

Comment (dyladan) [reply examples/dns/package.json:33]: I was going to cut a 0.14.0 release after this PR

File: detectors/node/opentelemetry-resource-detector-github/package.json (hunk 1 above)
Comment (dyladan) [reply detectors/node/opentelemetry-resource-detector-github/package.json:64]: it should only depend on resources anyways. good catch

File: examples/dns/package.json (hunk 2 above)
Comment (dyladan) [reply examples/dns/package.json:33]: devDependencies are pinned so the CI is faster. prod dependencies are ranged so customers can receive important bugfixes without manual intervention.

File: detectors/node/opentelemetry-resource-detector-github/package.json (hunk 1 above)
Comment (dyladan) [reply detectors/node/opentelemetry-resource-detector-github/package.json]: devDependencies are pinned so the CI is faster. prod dependencies are ranged so customers can receive important bugfixes without manual intervention.

Comment (dyladan) [reply detectors/node/opentelemetry-resource-detector-github/package.json]: devDependencies are pinned so the CI is faster. prod dependencies are ranged so customers can receive important bugfixes without manual intervention.

**Review by vmarchaud — APPROVED** [review]
Not really linked to this PR but i just remembered that both koa and hapi plugins were named with `instrumentation` even though there were based on the plugin system. It will introduces hard breaking changes for those who directly imported them

**Review by obecny — COMMENTED** [review]
looks like many things has changed already with this

**Comment by dyladan** [comment]
> looks like many things has changed already with this

yeah it's quite behind now. working on a new version

**Comment by obecny** [comment]
as spoken with @dyladan closing this PR and will prepare a new one

**Pull request closed without merging**